### ISSUE TITLE
Will never execute twice

### ISSUE THREAD

**Issue by xspyhack** [sunshinejr/RxPermission#6]
The `subscribeNext` closure will not execute twice at application's life cycle.
If this code is written in a UIViewController when second time appeared the label.text will be empty.

``` swift
Permission
            .LocationAlways
            .rx_permission
            .subscribeNext { status in
                print(status)
                self.label.text = "Status: \(status)"
            }
            .addDisposableTo(disposeBag)
```

**Comment by sunshinejr** [comment]
Hey @xspyhack, I was thinking similar you are but it is actually how [Permission](https://github.com/delba/Permission/pull/17) works. Basically it will trigger once after you request it. You might follow the example @ivanbruel suggested in #2, where you can `flatMap` the sequence every time you want the action. It is available in 0.2 version so you can take a look at tell me if there is anything we need more. Thanks! 🐼

**Comment by xspyhack** [comment]
It makes sense! Thanks!
